server/documents/modules/dropdown.html.eco

```diff
@@ -3072,7 +3072,7 @@ themes      : ['Default', 'GitHub', 'Material']
           <td>Recreates dropdown menu from passed values. <code>values</code> should be an object with the following structure: <code>{ values: [ {value, text, name} ] }</code>.</td>
         </tr>
           <td>change values (values)</td>
-          <td>Changes dropdown to use new values</td>
+          <td>Changes dropdown to use new values. <code>values</code> structure: <code>{ [ {value, text, name} ] }</code>.</td>
```

Comment:
According to your (correct) example, values is an array, not an object, so please remove the outer curly braces
```suggestion
          <td>Changes dropdown to use new values. <code>values</code> structure: <code>[ {value, text, name} ]</code>.</td>
```